src/marqo/README.md

```diff
@@ -37,7 +37,7 @@ __Notes__:
 - This is for marqo-os (Marqo OpenSearch) running locally. You can alternatively set
 `OPENSEARCH_URL` to  a remote Marqo OpenSearch cluster 
 - To find the absolute path to the `marqo/src` directory, `cd` into the `marqo/src` directory and run `pwd` in your terminal.
-- If python library errors occur, try `root/.../marqo/src` instead of `~/.../marqo/src`
+- If python library errors occur, try `/root/.../marqo/src` instead of `~/.../marqo/src`
```

Comment:
Could we call say "try the absolute path"? And typically `root` isn't part of the absolute path. This would be more appropriate: 


try `/.../marqo/src` instead of `~/.../marqo/src`

Reply:
git clone is suggested to use ssh instead of HTTPS because I have issues when trying to push to the repo using HTTPS (feature is no longer supported). This could just be a problem on my end though, I can scrap it.